libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -271,6 +271,15 @@ const AP_Param::GroupInfo AC_PosControl::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("_ANGLE_MAX", 7, AC_PosControl, _lean_angle_max, 0.0f),
 
+    // @Param: _MAX_LEASH
+    // @DisplayName: Position Control Leash Length Max
+    // @Description: Maximum position correction for velocity command.  Set to zero to use default calculate _leash value
```

Comment:
Would this be better as "deviation" rather than "correction"?

Reply:
Yeah I think that is a good idea. Ill change that